tests/utils.sh

```diff
@@ -295,6 +295,11 @@ function mk_fake_kw_env()
 # @result_to_compare A raw output from the string
 function compare_command_sequence()
 {
+  # colors to use to print failure
+  local color_none='\033[0m'
+  local color_red='\033[1;31m'
+  local color_green='\033[1;32m'
```

Comment:
You already declared these variables in the `assert_equals_helper`; it is time to make them read-only and global to this file to avoid code duplication.